src/compiler/compile/nodes/InlineComponent.ts

```diff
@@ -155,11 +157,24 @@ export default class InlineComponent extends Node {
 		}
 
 		this.children = map_children(component, this, this.scope, children);
+
+		this.validate();
 	}
 
 	get slot_template_name() {
 		return this.attributes.find(attribute => attribute.name === 'slot').get_static_value() as string;
 	}
+
+	validate() {
+		const label_has_associated_control_rule_options = this.component.compile_options.a11y?.rules?.['label-has-associated-control'];
```

Comment:
I think CI should fail because Node < 14 doesn't have optional chaining.
But why CI doesn't fail...???

Reply:
i think the test is testing on the build file, which has a output target es2017